packages/core/src/index.html

```diff
@@ -357,5 +368,47 @@
             })(window, document, [].slice);
         </script>
         <!-- End LiveChat script -->
+
+        <script>
```

Comment:
can we add content of this script tag in a js file and include the file in the script tag?